package.json

```diff
@@ -109,10 +107,20 @@
     "packages/*",
     "build-settings"
   ],
-  "pre-commit": [
-    "lint",
-    "git-add-snapshot-tests",
-    "test:fast",
-    "pretty-quick"
-  ]
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.md": [
```

Comment:
Ohh - this is really nice!

Reply:
Err - will this run `gen-shapshot-tests` once for all files or once for every file that was changed?

Reply:
This should run over all files since I'll need to make changes to the script being run to accept a list of files.